gsa/src/web/pages/page.js

```diff
@@ -116,9 +114,9 @@ const Page = props => {
 };
 
 Page.propTypes = {
-  gmp: PropTypes.gmp.isRequired,
+  children: PropTypes.func.isRequired,
 };
```

Comment:
I guess the proptypes can be dropped completely because children is validated automatically and shouldn't be a function here